cflat.md

```diff
@@ -0,0 +1,100 @@
+> [name=Arbie, Yin, Guo, Ashima]
+
+## Information about the project
+
+### What is compiler explorer?
+
+Compiler Explorer is an interactive compiler exploration website. Edit code in C, C++, C#, F#, Rust, Go, D, Haskell,
+Swift, Pascal, ispc, Python, Java, or any of the other 30+ supported languages, and see how that code looks after being
+compiled in real time. Multiple compilers are supported for each language, many different tools and visualizations are
+available, and the UI layout is configurable (thanks to GoldenLayout).
+
+### What is our project is about?
+
+This project focuses on enhancing Compiler Explorer (CE) by adapting it to support in-browser compilers. The primary
+objectives include restructuring the backend by removing the original CE server and integrating WebAssembly (Wasm) code
+for compiler execution. Additionally, the project aims to add a new feature of running different compiler passes so that
+users can better understand the compiler process. Furthermore, the project implements another functionality to load
+different compilers.
+
+## Build process
+
+### installation
+
+```bash=
+pip install node
```

Comment:
These instructions seem to be about installing Compiler Explorer itself. Perhaps if you plan on sharing this with other cflat users it can go in the instructions for them, but putting it here seems unusual. The main README.md has some instructions already for this. This also only works on Linux systems that have `pip` installed approparitely.